packages/monorepo-tools/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@mongodb-js/monorepo-tools",
+  "description": "MongoDB scripts for managing devtools monorepos",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared",
+  "version": "1.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-shared.git"
+  },
+  "license": "SSPL",
+  "files": [
+    "src",
+    "!src/**/*.spec.js",
+    "!src/fixtures"
+  ],
+  "main": "dist/index.js",
```

Comment:
Copy/pasted this from bump-monorepo-packages, but now not sure this is right. What's making dist/?

Reply:
I'll just convert all this type typescript.